packages/admin/admin/src/mui/mainNavigation/Context.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentType, createContext, Dispatch, FunctionComponent, SetStateAction } from "react";
+
+export interface MainNavigationContextValue {
```

Comment:
TypeScript supports both a type and a variable with the same name, so we don't need the Value suffix.